### PR TITLE
Refresh automotive page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Refresh automotive page

## QA

- Go to [https://ubuntu-com-12039.demos.haus/automotive](https://ubuntu-com-12039.demos.haus/automotive)
- Compare page against [copy doc](https://docs.google.com/document/d/10jSLdwz2Ks7gnEMXlKWzrpsY6TQFxISUgnQo4y3yOrA/edit#)

## Issue / Card

Fixes [#5934](https://github.com/canonical/web-squad/issues/5934)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @minkyngkm, can we update a few things please:

- Can you make the images in the strips bigger please, the [current page](https://ubuntu.com/automotive) is a good guide. Compare "autonomous cars" for example
- Make the first logo strip --light please
- Add a border hr to the top of the final logo strip

I guess we will need the final hero image before this goes live.

Thanks

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for reviewing! Yep, I'm waiting for the final hero image. Other than that, I made some updates based on your suggestions. Hopefully it looks better now! :)

**Comment by minkyngkm** [comment]
Hi @spencerbygraves! As demo is not working, could I give you a screenshot of the page?

Boost your automotive business with Ubuntu | Ubuntu (2022-09-22 08-45-29)

**Comment by spencerbygraves** [comment]
Thanks for updating @minkyngkm, looks good.

File: templates/automotive/index.html [hunk 1]
```diff
@@ -156,160 +249,147 @@ <h2 class="p-logo-section__title u-align--center">Accelerating on Ubuntu</h2>
...
   <div class="row u-equal-height">
     <div class="col-6">
-      <h2 class="p-heading--5">Over-the-air updates</h2>
-      <p class="p-heading--2">OTA? Compressed. Transactional. Signed. Simple.</p>
+      <h3 class="p-heading--2">Compressed, transactional, signed and simple OTA</h3>
       <p>Update any system, any device, any day. Future-proof your fleet.</p>
-      <p>Full OS containerisation means transactional, atomic updates. Data snapshot backup and restore. Signed software integrity. <a href="/engage/snap-deltas">Compressed delta updates</a>.</p>
-      <p><a href="/engage/snapd-whitepaper">Complete control</a>. No delayed patches, no partial updates, no unauthorised ECU software.</p>
+      <p>Full OS containerisation means transactional, atomic updates. Data snapshot backup and restore. Signed software integrity. Compressed delta updates.</p>
+      <p>Complete control. No delayed patches, no partial updates, no unauthorised ECU software.elivers the highest multi-cloud performance for AI/ML.</p>
```
Comment (petesfrench) [inline templates/automotive/index.html:392]:
```suggestion
      <p>Complete control. No delayed patches, no partial updates, no unauthorised ECU software.</p>
```

File: templates/automotive/index.html [hunk 2]
```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
...
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">best developers choose Ubuntu</a>.</li>
-        <li class="p-list__item is-ticked">The biggest pool of talent. The broadest range of tools.  The community that counts.</li>
-        <li class="p-list__item is-ticked">One company stands behind it all &mdash; Canonical.</li>
-      </ul>
+  <div class="u-fixed-width">
+    <h2>Preferred by developers</h2>
+    <p>Ubuntu is the <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">preferred OS for developers</a>, at 66% of working professionals &ndash; and growing.</p>
+    <p>Empower your team and attract the best talent.</p>
+    <p><a href="/automotive/developer">Read more about the Ubuntu developer experience&nbsp;&rsaquo;</a></p>
```
Comment (petesfrench) [inline templates/automotive/index.html:637]:
```suggestion
    <p><a href="/automotive/developers">Read more about the Ubuntu developer experience&nbsp;&rsaquo;</a></p>
```
Will this /automative/developers be live before this is merged, if not we should remove this link for now

File: templates/automotive/index.html [hunk 3]
```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
...
+            url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
+            alt="whitepaper",
+            width="171",
+            height="150",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">whitepaper</h4>
```
Comment (petesfrench) [inline templates/automotive/index.html:660]:
```suggestion
          <h4 class="p-heading-icon__title">Whitepaper</h4>
```

File: templates/automotive/index.html [hunk 4]
```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
...
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Announcement</h4>
+        </div>
+      </div>
+      <p>Stay tuned!</p>
```
Comment (petesfrench) [inline templates/automotive/index.html:703]: You might want to get another opinion on this, but to me this feels a little off-brand.

File: templates/automotive/index.html [hunk 5]
```diff
@@ -5,44 +5,65 @@
...
       {{
         image(
-        url="https://assets.ubuntu.com/v1/86148922-Automotive-in-car-cockpit-cropped.svg",
+        url="https://assets.ubuntu.com/v1/2bd31151-Automotive-in-car-cockpit-sized.svg",
         alt="",
-        width="605",
-        height="380",
+        height="250",
+        width="429",
         hi_def=True,
```
Comment (petesfrench) [inline templates/automotive/index.html:53]:
```suggestion
        hi_def=True,
        loading=lazy,
```

File: templates/automotive/index.html [hunk 6]
```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
...
       {{
         image(
-        url="https://assets.ubuntu.com/v1/b2b4d912-community-support-thin.svg",
+        url="https://assets.ubuntu.com/v1/22007291-DevOps.svg",
         alt="",
-        height="120",
-        width="96",
+        height="200",
+        width="350",
         hi_def=True,
```
Comment (petesfrench) [inline templates/automotive/index.html:545]:
```suggestion
        hi_def=True,
        loading=lazy
```

File: templates/automotive/index.html (hunk 4 above)
Comment (minkyngkm) [reply templates/automotive/index.html]: I will just remove this column then!

**Review by petesfrench — APPROVED** [review]
LGTM :+1:

**Pull request merged**